test.js

```diff
@@ -447,3 +450,60 @@ test('alias - require previousId and userId', t => {
     })
   })
 })
+
+test.cb('e2e', t => {
+  const id = uid(16)
+
+  const testFn = function () {
+    var axiosClient = axios.create({
+      baseURL: 'https://api.runscope.com',
+      timeout: 10 * 1000,
+      headers: { 'Authorization': 'Bearer ' + process.env.RUNSCOPE_TOKEN }
+    })
+
+    retries(axiosClient, { retries: 3 })
+
+    axiosClient.get('buckets/zfte7jmy76oz/messages?count=10')
```

Comment:
What is “zfte7jmy76oz”?

Reply:
It's the ID for our Runscope bucket. I'll link to it here.

Reply:
Ahh ok, the a link would be great. 👍 

Maybe a comment too?